data/geany-3.0.css

```diff
@@ -1,10 +0,0 @@
-@import "geany.css";
-
-/* make close button on the editor's tabs smaller */
-#geany-close-tab-button {
-	-GtkWidget-focus-padding: 0;
-	-GtkWidget-focus-line-width: 0;
-	-GtkButton-default-border: 0;
-	-GtkButton-default-outside-border: 0;
-	-GtkButton-inner-border: 0;
-}
```

Comment:
Okay, you can't apparently, as we don't seem to set the `GeanyDialog` name on it.  So well, ok I guess, point taken.  Not that I can imagine any single good reason why one would want to style differently the very same dialog, but well.